README.md

```diff
@@ -0,0 +1,271 @@
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add webpack**
+- **Add react components**
+- **Display a random greeting**
+- **Create API endpoint for the greeting**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+gem install ruby
+```
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Luiscarlosvd/hello-rails-react.git
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd hello-rails-react
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+npm run build
+rails s
+```
+**Note:** If there are any errors consider adding your necesary username and password in `database.yml` for setting up your local enviroment and use the following comand:
+
+```
+rails db:create db:migrate
+rails s
+```
+
+<!-- ### Run tests
+
+To run tests, run the following command:
+
+```
+rails tailwindcss:watch
+control + c
+rspec spec 
+```
+-->
+<!-- ### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Luis Valera**
+
+- GitHub: [@Luiscarlosvd](https://github.com/Luiscarlosvd)
+- Twitter: [@Luiscarlosvd29](https://twitter.com/Luiscarlosvd29)
+- LinkedIn: [Luis Valera](https://www.linkedin.com/in/luis-valera-6a5749267/)
+
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve UI**
+- [ ] **Add more functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
![image](https://github.com/Luiscarlosvd/hello-rails-react/assets/93039359/078c1d72-ee67-4b93-8548-bd61defbb3bf)


- [ ] It would be nice if you fix this issue link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍